service/service.go

```diff
@@ -354,32 +355,32 @@ func (s *Service) newEngine() error {
 	}
 	var ng engine.Engine
 
+	options := etcdng.Options{
+		CaFile:              s.options.EtcdCaFile,
+		CertFile:            s.options.EtcdCertFile,
+		KeyFile:             s.options.EtcdKeyFile,
+		Consistency:         s.options.EtcdConsistency,
+		SyncIntervalSeconds: s.options.EtcdSyncIntervalSeconds,
+		Username:            s.options.EtcdUsername,
+		Password:            s.options.EtcdPassword,
+		InsecureSkipVerify:  s.options.EtcdInsecureSkipVerify,
+		EnableTLS:           s.options.EtcdEnableTLS,
+		Debug:               s.options.EtcdDebug,
+		Box:                 box,
+	}
+
 	if s.options.EtcdApiVersion == 3 {
-		ng, err = etcdv3ng.New(
+		ng, err = v3.New(
```

Comment:
In this context `v3` is not descriptive enough. I think we better use import aliases (e.g. etcd3ng) or something like that.